### DISCUSSION TITLE
能否支持 相册创建文件夹上传？

### DISCUSSION THREAD

**Discussion by imrelax** [Molunerfinn/PicGo#800] [Ideas]
图片太多了 都在一个相册里面管理太麻烦了。能否支持到文件夹 管理？做一些文件夹分类

然后关于图片不能远程删除的问题。
部分OSS 是不支持远程删除的。但是部分不是支持嘛。
对于支持的能不能比如做个勾选之类的。支持远程删除。

**Comment by Molunerfinn** [comment]
精力有限，有些能力可以让插件实现的就让插件实现了。另外也欢迎开发者一起参与进来贡献代码。目前这个项目99%的代码都还是我在写，这对于已经工作的人来说其实很耗费精力，也希望谅解

远程删除这个功能，已经有开放删除钩子给插件实现。比如github-plus这个插件就实现了同步删除的能力。
